std_msgs/Header header
string[] colors
geometry_msgs/Point[] positions

